olive/passes/onnx/inc_quantization.py

```diff
@@ -272,6 +271,17 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
         config.update(get_external_data_config())
         return config
 
+    @staticmethod
+    def _data_configs() -> Dict[str, PassDataConfigParam]:
+        return {
+            "data_config": PassDataConfigParam(
```

Comment:
better to "calibration_data_config"? It may be clearer for user.

Reply:
Yes, I considered changing the name but didn’t do it in this PR since I will have to update the examples haha